pkg/constants/constants.go

```diff
@@ -220,6 +221,31 @@ var (
 	}
 )
 
+var (
+	//RawIngressNamePrefix ...
+	RawIngressNamePrefix = KFServingName + "-ingress"
```

Comment:
Do we need the prefix? dns has length limit for 63

Reply:
ok